feature-pack/src/main/resources/modules/system/layers/base/com/redhat/gss/redhat-support-lib-java/main/module.xml

```diff
@@ -0,0 +1,18 @@
+<module xmlns="urn:jboss:module:1.0" name="com.redhat.gss.redhat-support-lib-java">
+    
+    <resources>
+        <artifact name="${com.redhat.gss:redhat-support-lib-java}"/>
+    </resources>
+
+    <dependencies>
+        <module name="javax.api"/>
+        <module name="javax.ws.rs.api" services="import"/>
+        <!-- <module name="javax.net.ssl" slot='main'/> -->
+        <module name="org.apache.commons.net" slot='main'/>
```

Comment:
that is fine, enforcement works as it should :) 
you need to add <exclude> for all transitive dependencies of the module